components/Form/Form.js

```diff
@@ -60,6 +62,16 @@ export default function Form({
     setCollectionId(event.target.value);
   }
 
+  function handleLanguageChangeQuestion(event) {
+    setLanguageQuestion(event.target.value);
+    card.languageQuestion = event.target.value;
+  }
+
+  function handleLanguageChangeAnswer(event) {
+    setLanguageAnswer(event.target.value);
+    card.languageAnswer = event.target.value;
```

Comment:
Again, we seem to be mutating the props here.

Reply:
fixed.